### DISCUSSION TITLE
Brighter for DotNet Framework (4.7)

### DISCUSSION THREAD

**Discussion by mhsayan** [BrighterCommand/Brighter#2414] [Q&A]
As I am new to brighter, I'm facing some issues to use it with DotNet Framework. All of these examples are based on dependency injection. But my project does not use any dependency injection. So, it's getting harder than ever for me to implement this Awesome library with my project.

This will be very much helpful for me to get any suggestions/examples to use it without any dependency injection.

**Comment by holytshirt** [comment]
Hi @mhsayan would some examples using Castle Windsor on net4.7 be useful?
